bin/tessel-2.js

```diff
@@ -183,6 +185,10 @@ makeCommand('wifi')
     metavar: 'PASSWORD',
     help: 'Set the password of the network to connect to'
   })
+  .option('off', {
+    flag: true,
+    help: 'Disable the wireless network'
```

Comment:
Would it be more accurate to say "Disable the wireless network connection" ? This is more picky, semantics than anything.